### DISCUSSION TITLE
iPadOS 16、macOS Ventura 正式版推出日期確定，這些是可以升級的型號

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#806] [General]
ipados 跟macos也要迎來更新了，這次更新的功能似乎沒有甚麼讓我驚艷的功能，只期望外觀部分也能像ios16有更多自訂性😍
[文章連結](https://www.kocpc.com.tw/archives/464696)

**Comment by Pietra1226** [comment]
其實蘋果的外觀我都蠻滿意的，他們抓的顏色都真的很美

**Comment by evanchou1204** [comment]
之前在脫口秀聽到一個段子，熊仔的新專輯是pro，靈感來自iphone，都沒進步但是可以賣很貴。果粉：真香。
